src/abstractdataframe/reshape.jl

```diff
@@ -423,13 +441,11 @@ function unstack(df::AbstractDataFrame, rowkeys, colkey::ColumnIndex,
     else
         df_op = df
```

Comment:
Given that we have this, maybe do `values_out = values` here instead of `values = values_out` above, and use `values_out` below?

Reply:
OK